prolog:
  global_stack: 256
  local_stack: 256
  trail_stack: 256
  
  num_engines: 4
  executable: /usr/bin/swipl
  
